src/commands/initProjectForVSCode/InitVSCodeStep/ScriptInitVSCodeStep.ts

```diff
@@ -15,32 +17,36 @@ import { InitVSCodeStepBase } from './InitVSCodeStepBase';
  * Base class for all projects based on a simple script (i.e. JavaScript, C# Script, Bash, etc.) that don't require compilation
  */
 export class ScriptInitVSCodeStep extends InitVSCodeStepBase {
-    protected requiresFuncExtensionsInstall: boolean = false;
+    protected useFuncExtensionsInstall: boolean = false;
 
     protected getTasks(): TaskDefinition[] {
         return [
             {
                 type: func,
                 command: hostStartCommand,
                 problemMatcher: funcWatchProblemMatcher,
-                dependsOn: this.requiresFuncExtensionsInstall ? extInstallTaskName : undefined,
+                dependsOn: this.useFuncExtensionsInstall ? extInstallTaskName : undefined,
                 isBackground: true
             }
         ];
     }
 
     protected async executeCore(context: IProjectWizardContext): Promise<void> {
-        if (context.version === FuncVersion.v2) { // no need to check v1 or v3+
-            try {
+        try {
+            const extensionsCsprojPath: string = path.join(context.projectPath, 'extensions.csproj');
+            if (await fse.pathExists(extensionsCsprojPath)) {
```

Comment:
Should we alert or prompt the user telling them that csproj files aren't necessary anymore?  Not sure how much we want to get users off this path.

Reply:
There are still a few advanced scenarios where the csproj file is better than bundles, but yeah a prompt might make sense for the majority of users. I added `hasExtensionsCsproj` as a telemetry property which should help us decide if it's worth the work